demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -786,3 +787,22 @@ def is_valid_rule_suffix(self, rule_type: str) -> bool:
                 return False
 
         return True
+
+    def is_line_ends_with_dot(self, dict_to_test: dict, arg_field: str):
+        line_with_missing_dot = ""
+        for arg in dict_to_test.get(arg_field, []):
+            stripped_description = arg.get("description", "").strip('"').strip("'")
+
+            if (
+                stripped_description
+                and not stripped_description.endswith(".")
+                and not is_string_ends_with_url(stripped_description)
+            ):
+                line_with_missing_dot += f"The argument {arg.get('name')} description should end with a period.\n"
+        for output in dict_to_test.get("outputs") or []:
+            stripped_description = output.get("description", "").strip('"').strip("'")
```

Comment:
```suggestion
            stripped_description = output.get("description", "").strip('"\'')
```

Reply:
Not sure this is correct.. I don't want to strip the sequence "', I want to strip either " or ' because I don't know how the field will look like..